doc/source/basicgameplay/companies.rst

```diff
@@ -7,6 +7,16 @@ these companies, you can apply for jobs.
 
 Working a job lets you earn money, experience, and reputation with that company.
 
+While working for a company, you can click "Do something else simultaneously" be able 
+to do things while you continue to work in the background. There is a 20% penalty the 
+related gains. Clicking the "Focus" button under the overview will return you to the 
+current work.   
+
+Reputation is required to apply for a promotion. This reputation is not counted towards
+your career until the shift ends, either due to the time spent or clicking the 
+"Stop Working" button. For most jobs there is a penalty of 50% of the reputation gained
+if you stop your shift early. 
```

Comment:
You can add some info that backdooring reduces this penalty

Reply:
Does that belong in the documentation, or is that something better left vague and found by the players?

Reply:
TIL

Reply:
> You can add some info that backdooring reduces this penalty

I'm not sure if that's public knowledge or more discovered knowledge. 

I vaguely hinted at that in the backdoor changes in the hacking documentation.

Reply:
When I found out I kinda felt like - I would have liked to have read that in the docs - but I get your point

Reply:
In #3010, I mention that backdoors reduce penalties. So it's in there now.